Makefile

```diff
@@ -536,15 +536,28 @@ k3d-destroy-cluster:
 logs-dir-setup:
 	@mkdir -p tests/logs
 
-.PHONY: deploy-fluent-bit
-deploy-fluent-bit: kubectl
-	$(KUBECTL) apply -f tests/fluent-bit/
-	$(KUBECTL) -n kube-system rollout status daemonset fluent-bit
-
 .PHONY: logs-dir-cleanup
 logs-dir-cleanup:
 	@rm -rf tests/logs/*
 
+.PHONY: deploy-fluent-bit
+deploy-fluent-bit: kubectl apply-fluent-bit status-fluent-bit
```

Comment:
As a follow-up: Should we just make this part of `make {k3d,kind}-setup`? (wondering what utility there is in keeping makefile targets for fluent-bit)

Reply:
It is already part of `deploy`. 

Separate targets are mostly for testing, and in general, "more targets" is better than "less targets" - same logic as for functions in normal code, more flexibility when working with repo locally. WDYT?